### PR TITLE
Master ⚡ , change slight bits of code, and edit readme.

### PR DESCRIPTION
**Pull request by qaiik**
Before sending the pull request, make sure that:

- [ ] `gulp build` runs without issues.
- [ ] if you develop a catmod, a test project and docs are also present. Put a test project into a catmod's directory.
- [ ] if it is a work-in-progress, explicitly mark it as such in the title with the `WIP: ` prefix.
- [ ] The title of the pull request has a suitable emoji from [the Comigoji guide](https://comigo.gitlab.io/comigoji/#gitmoji). The title *will* look broken on GitHub, but it streamlines further merging and makes writing a pretty changelog easier.

--------------------------------------------------------

Closes #(issue's index).

**Changes proposed in this pull request:**

Basically the title, change some of the readme stuff, and like slight bits of code (maybe)
**Help wanted:** (optional, e.g. a request for GUI revamp)

no help wanted

**Ping @CosmoMyzrailGorynych**

### CODE REVIEW

**Comment by CosmoMyzrailGorynych** [comment]
Please pull from upstream before doing anything. You are doing changes from a version that is literally 80+ commits behind develop, not speaking of v1.x.

**Comment by qaiik** [comment]
k yeah sorry
